lightway-core/src/connection.rs

```diff
@@ -738,6 +738,11 @@ impl<AppState: Send> Connection<AppState> {
         Ok(auth_handle.expired())
     }
 
+    /// Update WolfSSL Session to use the new outside IO Callback
+    pub fn set_new_outside_io(&mut self, new_io: OutsideIOSendCallbackArg) {
```

Comment:
```suggestion
    pub fn set_outside_io(&mut self, new_io: OutsideIOSendCallbackArg) {
```
Let's makes this consistent with other apis.